grid/cell.py

```diff
@@ -134,6 +134,7 @@ def triangular_number(n: int) -> int:
 
 
 class Position(NamedTuple):
```

Comment:
I wonder if `Offset` is a better name

Reply:
I'm inclined to not rename `Position` to avoid confusion right now.